outputs.tf

```diff
@@ -1,8 +1,18 @@
-##############################################################################
+########################################################################################################################
 # Outputs
-##############################################################################
+########################################################################################################################
 
-#output "myoutput" {
-#  description = "Description of my output"
-#  value       = "value"
-#}
+output "enterprise_hierarchy_depth_0" {
+  value       = module.dynamic_values.enterprise_hierarchy_depth_0
+  description = "Enterprise input for depth 0"
+}
+
+output "enterprise_hierarchy_depth_1" {
+  value       = module.dynamic_values.enterprise_hierarchy_depth_1
+  description = "Enterprise input for depth 1"
+}
+
+output "enterprise_hierarchy_depth_2" {
+  value       = module.dynamic_values.enterprise_hierarchy_depth_2
+  description = "Enterprise input for depth 2"
+}
```

Comment:
Not sure these outputs make sense for consumers. Can we combine the outputs so there is one output which includes all accounts + account groups that were created?

Reply:
fixed it